tools/meteor-services/deploy.js

```diff
@@ -368,7 +370,8 @@ var bundleAndDeploy = function (options) {
   var preflight = authedRpc({
     site: site,
     preflight: true,
-    promptIfAuthFails: promptIfAuthFails
+    promptIfAuthFails: promptIfAuthFails,
+    qs: options.rawOptions
```

Comment:
Oh, OK I read it more. I guess we do the preflight first to avoid having to wait to bundle to crap out. So this is good.